sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/ConflictResolutionPolicy.java

```diff
@@ -15,49 +14,49 @@
 
 /**
  * Represents the conflict resolution policy configuration for specifying how to resolve conflicts
- * in case writes from different regions result in conflicts on documents in the collection in the Azure Cosmos DB
+ * in case writes from different regions result in conflicts on items in the container in the Azure Cosmos DB
```

Comment:
please add a link to https://docs.microsoft.com/en-us/azure/cosmos-db/conflict-resolution-policies here.